src/NSwag.Sample.NETCore21/Controllers/PetController.cs

```diff
@@ -20,6 +20,7 @@ public class PetController : ControllerBase
         [HttpPost]
         [Consumes("application/json")]
         [Produces("application/json")]
+        [ProducesResponseType(typeof(void), (int)HttpStatusCode.NoContent)]
```

Comment:
Might be nit-picky, but you could use `Microsoft.AspNetCore.Http.StatusCodes.` here to avoid casting to int.